### PR TITLE
LTD-5015 Change 'Item list codes' to 'Control list entry'

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Content change - 'Item list codes' to 'Control list entry' for headers in denials list and denial details pages

[LTD-5015](https://uktrade.atlassian.net/browse/LTD-5015)

[LTD-5015]: https://uktrade.atlassian.net/browse/LTD-5015?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: caseworker/templates/external_data/denial-detail.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h1 class="govuk-heading-l">Denial details record</h1>
                 <td class="govuk-table__cell">{{ denial.country }}</td>
             </tr>
             <tr class="govuk-table__row app-table__row">
-                <th scope="row" class="govuk-table__header">Item List Codes</th>
+                <th scope="row" class="govuk-table__header">Denial control list entry</th>
                 <td class="govuk-table__cell">{{ denial.item_list_codes }}</td>
```
Comment (saruniitr) [inline caseworker/templates/external_data/denial-detail.html:64]: list codes are the same as Control list entries?

Comment (markj0hnst0n) [reply caseworker/templates/external_data/denial-detail.html:64]: Yes [this PR](https://github.com/uktrade/lite-frontend/pull/1971) and [this one](https://github.com/uktrade/lite-api/pull/2000) deal with changing the variables to this name.  This PR probably should have come after those.

Denial control list entry is a more descriptive name and it's attempting to make clear that this is a control list entry related to denials only as advised by the business.

**Review by depsiatwal — APPROVED** [review]
lgtm

File: caseworker/templates/external_data/denial-detail.html (hunk 1 above)
Comment (depsiatwal) [reply caseworker/templates/external_data/denial-detail.html:64]: @saruniitr yeah this is the story raised by Collin item_list_code are similar to what we call CLE

**Comment by markj0hnst0n** [comment]
folded into another PR [here](https://github.com/uktrade/lite-frontend/pull/1971)

**Pull request closed without merging**